ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx

```diff
@@ -61,78 +54,45 @@ const OperationIcon = ({ operation }: OperationComponentProps) => {
     case TransactionOperationTypeEnum.TRANSACTION:
       if (operation.channel === ChannelEnum.QRCODE) {
-        return <Icon name={"productIOAppBlueBg"} color="bluegreyLight" />;
+        return <Icon name={"merchant"} color="bluegreyLight" />;
       }
-      return getCardLogoComponent(operation.brand);
+      return operation.brand;
     case InstrumentOperationTypeEnum.ADD_INSTRUMENT:
     case InstrumentOperationTypeEnum.DELETE_INSTRUMENT:
-      return getCardLogoComponent(operation.brand);
+      return operation.brand;
     default:
       return null;
   }
 };
 
-const getDiscountInitiativeTransactionLabel = (
+const getDiscountInitiativeTransactionStatus = (
   operation: TransactionOperationDTO
-) => {
+): ListItemTransactionStatus => {
   switch (operation.status) {
     case TransactionStatusEnum.CANCELLED:
-      return (
-        <IOBadge
-          color="red"
-          variant="solid"
-          text={I18n.t(
-            "idpay.initiative.operationDetails.discount.labels.CANCELLED"
-          )}
-        />
-      );
+      return "cancelled";
     case TransactionStatusEnum.REWARDED:
     case TransactionStatusEnum.AUTHORIZED:
-      return (
-        <H4>{`-${formatAbsNumberAmountOrDefault(operation.amount)} €`}</H4>
-      );
+      return "success";
   }
 };
 
-const OperationAmountOrLabel = ({ operation }: OperationComponentProps) => {
+const getTransactionAmountLabel = (operation: OperationListDTO) => {
   switch (operation.operationType) {
     case TransactionOperationTypeEnum.TRANSACTION:
-      if (operation.channel === ChannelEnum.QRCODE) {
-        return getDiscountInitiativeTransactionLabel(operation);
-      }
-
-      return (
-        <H4>{`-${formatAbsNumberAmountOrDefault(operation.accrued)} €`}</H4>
-      );
+      return `-${formatAbsNumberAmountOrDefault(operation.accrued)} €`;
     case TransactionOperationTypeEnum.REVERSAL:
-      return (
-        <H4>{`+${formatAbsNumberAmountOrDefault(operation.accrued)} €`}</H4>
-      );
+      return `+${formatAbsNumberAmountOrDefault(operation.accrued)} €`;
     case RefundOperationTypeEnum.PAID_REFUND:
-      return (
-        <H4 color="greenLight">
-          {`${formatAbsNumberAmountOrDefault(operation.amount)} €`}
-        </H4>
-      );
+      return `${formatAbsNumberAmountOrDefault(operation.amount)} €`;
     default:
-      return null;
+      return "";
   }
 };
 
 const TimelineOperationListItem = (props: TimelineOperationListItemProps) => {
   const { operation, onPress } = props;
 
-  const isQrCodeTransaction =
-    operation.operationType === TransactionOperationTypeEnum.TRANSACTION &&
-    operation.channel === ChannelEnum.QRCODE;
-
-  const isCancelledQrCodeTransaction =
-    isQrCodeTransaction && operation.status === TransactionStatusEnum.CANCELLED;
-
-  const labelStyle: ViewStyle = isCancelledQrCodeTransaction
-    ? { opacity: 0.6 }
-    : {};
-
   const getOperationTitle = () => {
     switch (operation.operationType) {
```

Comment:
```suggestion
    switch (operation.operationType) {
case InstrumentOperationTypeEnum.ADD_INSTRUMENT:
        return I18n.t(
          "idpay.initiative.details.initiativeDetailsScreen.configured.operationsList.operationDescriptions.ADD_INSTRUMENT",
          {
            maskedPan: operation.maskedPan
          }
        );
```
to avoid missing I18n errors, we need to specify it, it probably went lost in all the changes 👍 
<img width="291" alt="Screenshot 2023-07-13 at 17 03 58" src="https://github.com/pagopa/io-app/assets/60693085/c8788e01-57ed-4eb9-a7ea-ecf74f55c76d">

Reply:
that said, seeing the designs the maskedPan is probably getting removed, so I'd ask @thisisjp about this